storm-core/src/storm.thrift

```diff
@@ -193,6 +193,7 @@ struct ExecutorStats {
   1: required map<string, map<string, i64>> emitted;
   2: required map<string, map<string, i64>> transferred;
   3: required ExecutorSpecificStats specific;
+  4: required double rate;
```

Comment:
Older client newer server works, newer client with older server brakes because rate is required behavior. I think this is acceptable and would hate to change a required param to optional to support this.